recipes/emsdk/all/test_example/conanfile.py

```diff
@@ -0,0 +1,28 @@
+import os
+from conan import ConanFile
+from conan.tools.cmake import CMakeToolchain, CMake, cmake_layout, CMakeDeps
+
+
+class EmsdkExampleRecipe(ConanFile):
+    name = "emsdk-example"
+    version = "1.0"
+    package_type = "application"
+    settings = "os", "compiler", "build_type", "arch"
+
+    def layout(self):
+        cmake_layout(self)
+
+    def generate(self):
+        deps = CMakeDeps(self)
+        deps.generate()
+        tc = CMakeToolchain(self)
+        tc.generate()
+
+    def build(self):
+        cmake = CMake(self)
+        cmake.configure()
+        cmake.build()
+
+    def test(self):
```

Comment:
This ``test()`` method never executes here in a regular recipe, it only executes in ``test_package`` recipes.